pkg/controller/nodelifecycle/scheduler/timed_workers_test.go

```diff
@@ -62,16 +65,16 @@ func TestExecuteDelayed(t *testing.T) {
 	})
 	now := time.Now()
 	then := now.Add(10 * time.Second)
-	queue.AddWork(NewWorkArgs("1", "1"), now, then)
-	queue.AddWork(NewWorkArgs("2", "2"), now, then)
-	queue.AddWork(NewWorkArgs("3", "3"), now, then)
-	queue.AddWork(NewWorkArgs("4", "4"), now, then)
-	queue.AddWork(NewWorkArgs("5", "5"), now, then)
-	queue.AddWork(NewWorkArgs("1", "1"), now, then)
-	queue.AddWork(NewWorkArgs("2", "2"), now, then)
-	queue.AddWork(NewWorkArgs("3", "3"), now, then)
-	queue.AddWork(NewWorkArgs("4", "4"), now, then)
-	queue.AddWork(NewWorkArgs("5", "5"), now, then)
+	queue.AddWork(NewWorkArgs("1", "1", metav1.TenantSystem), now, then)
```

Comment:
curious: what would happen if these were called using a regular tenant.

Reply:
That should work for a regular tenant. The manual test cases I gave in the PR description are for cases where the tenant are not system. 

 As we are going to have e2e testsfor kubectl, I did not addd multi-tenancy test cases for non-system tenant in this PR.